catalog/app/containers/Auth/messages.js

```diff
@@ -30,14 +30,44 @@ export default defineMessages({
     defaultMessage: 'Authenticating...',
   },
 
+  // SSO
+  ssoGoogleUse: {
+    id: `${scope}.SSO.Google.use`,
+    defaultMessage: 'Use Google Account',
+  },
+  ssoGoogleError: {
+    id: `${scope}.SSO.Google.error`,
+    defaultMessage: "Couldn't sign in with your Google account",
```

Comment:
`Unable to sign in with Google. <way to fix or pobable cause>`. All error messages should be `PROBLEM. SOLUTION.` what could cause this? admin not creating domain for this?

Reply:
notification renders the details string from google after this message